src/main.cc

```diff
@@ -15,15 +15,15 @@
 constexpr std::string_view USAGE_DETAILS{R"EOF(
 Global Options:
 
-   --verbose, -v                  Enable verbose output
```

Comment:
I think we shouldn't remove `-v` here, as it is still applicable as a verbosity flag in most other commands.